src/index.html

```diff
@@ -4,14 +4,174 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, user-scalable=no,
+      initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form onsubmit="onSubmit()" action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post">
+      <fieldset>
+        <legend>Personal information</legend>
+        <label for="Surname">
+          Surname:
+          <input
+            type="text"
+            name="Surname"
+            id = "Surname"
+            required
+          >
+        </label>
+        <label class="field-name" for="Name">
+          Name:
+          <input
+            type="text"
+            name="Name"
+            id="Name"
+            required
+          >
+        </label>
+        <label class="field-name" for="Age">
+          How old are You?:
+          <input
+            type="number"
+            name="Age"
+            id="Age"
+            min="12"
+            max="100"
+            value="12"
+            required
+          >
+        </label>
+        <label class="field-name" for="Date">
+          Full date of birth:
+          <input
+            type="date"
+            name="DateOfBirth"
+            id="Date"
```

Comment:
```suggestion
            name="Date"
            id="Date"
```
name та id прийнято робити однаковими